tests/cicd/ModelAdapterJenkinsfile

```diff
@@ -0,0 +1,93 @@
+pipeline {
+    agent {
+        label 'sr613'
+    }
+    stages {
+        stage('Prepare') {
+            steps {
+                script {
+                    def labels = ['sr613']
+                    def builders = [: ]
+                    for (x in labels) {
+                        def label = x
+                        builders[label] = {
+                            node(label) {
+                                cleanWs()
+                                checkout scm
+                                sh 'cd Dockerfile-ubuntu18.04 && docker build -t e2eaiok-pytorch120 . -f DockerfilePytorch120 && cd .. && yes | docker container prune && yes | docker image prune'
+                            }
+                        }
+                    }
+                    parallel builders
+                }
+            }
+        }
+        stage('Run model_adapter_unittest') {
+            agent {
+                label 'sr613'
+            }
+            steps {
+                sh 'docker run --rm --name test-ma-singlenode --privileged --network host --device=/dev/dri --shm-size="2g"  -v /mnt/DP_disk1/tlk:/home/vmagent/app/data  -v `pwd`:/home/vmagent/app -w /home/vmagent/app/test/cicd e2eaiok-pytorch120 /bin/bash -c ". /home/vmagent/app/tests/cicd/jenkins_model_adapter_unittest.sh"'
+            }
+        }
+        stage('Run model_adapter_DDP_baseline_test') {
+            agent {
+               label 'sr613'
+            }
+            steps {
+               sh 'docker run -t -d --name test-ma-multinode-baseline-r1 --privileged --network host --device=/dev/dri -v /mnt/DP_disk1/tlk:/home/vmagent/app/data  -v `pwd`:/home/vmagent/app -w /home/vmagent/app/e2eAIOK/e2eAIOK/ModelAdapter/src e2eaiok-pytorch120 /bin/bash -c ". /home/vmagent/app/tests/cicd/jenkins_model_adapter_basic_test_ddp.sh"'
+           }
```

Comment:
@XinyaoWa @zigzagcai , I saw in other jenkins file, we are using /home/vmagent/app/e2eaiok/e2eAIOK instead of /home/vmagent/app/e2eAIOK/e2eAIOK, can this path also align with @XinyaoWa @zigzagcai , I saw in other jenkins file, we are using /home/vmagent/app/e2eaiok/e2eAIOK?

Reply:
Updated